RoboVikes2016.yaml

```diff
@@ -891,7 +1052,7 @@ Properties:
     name: Export Subsystems, validators: null, value: true}
   Simulation World File: !!robotbuilder.data.properties.StringProperty {default: /usr/share/frcsim/worlds/GearsBotDemo.world,
     name: Simulation World File, validators: null, value: /usr/share/frcsim/worlds/GearsBotDemo.world}
-  Wiring file location: !!robotbuilder.data.properties.FileProperty {default: 'C:\Workspace2016',
+  Wiring file location: !!robotbuilder.data.properties.FileProperty {default: 'C:\Users\vandenrobotics\',
```

Comment:
Should it be this file path? This is for saving the file correct? @duanem  or @duane701 Which username should I@ you at?